fastlane/lib/fastlane/commands_generator.rb

```diff
@@ -86,6 +86,29 @@ def run
 
       always_trace!
 
+      command "validate-fastfile" do |c|
+        c.syntax = 'fastlane validate-fastfile /path/to/Fastfile'
+        c.option '--platforms STRING', String, 'Comma seperated list of platforms (ios, mac, android) of actions to validate (others will be skipped)'
+        c.action do |args, options|
+          if File.exist?(args.first)
+
+            plat = []
+            plat = options.platforms.split(",") if options.platforms
+            fl_parser = FastfileParser.new(content: File.read(args.first), filepath: args.first, name: "Fastfile", platforms: plat)
```

Comment:
This should look for the default Fastfile by default. Use `FastlaneFolder.fastfile_path` and make sure to test it with and without a Fastfile